src/wp-admin/update-core.php

```diff
@@ -269,6 +269,17 @@ function core_upgrade_preamble() {
 			__( 'https://wordpress.org/support/article/updating-wordpress/' )
 		);
 		echo '</p></div>';
+
+		if ( version_compare( $wp_version, '4.1.0', '<' ) ) {
+			echo '<div class="notice notice-warning inline"><p>';
+			printf(
+				/* translators: 1: Documentation on WordPress backups, 2: Documentation on updating WordPress. */
```

Comment:
It should be WordPress version.

```suggestion
				/* translators: 1: WordPress version, 2: Documentation on updating WordPress. */
```